pageserver/src/pgdatadir_mapping.rs

```diff
@@ -2102,9 +2102,9 @@ impl<'a> Version<'a> {
 //--- Metadata structs stored in key-value pairs in the repository.
 
 #[derive(Debug, Serialize, Deserialize)]
-struct DbDirectory {
+pub struct DbDirectory {
```

Comment:
The `pub`s in this file can all be `pub(crate)` because flow.rs is part of pageserver.

Reply:
https://github.com/neondatabase/neon/pull/9218/commits/cd1c11494ae534d3cda9d502ca5bfbd7dd7200e6